src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -761,6 +760,13 @@ public override void DeleteGeometryForIdentifier(string identifier, bool request
             DeleteGeometries(geometryModels, requestUpdate); 
         }
 
+        protected override void OnRenderPackagesUpdated(NodeModel node, IEnumerable<IRenderPackage> packages)
```

Comment:
If we only remove render packages if there are no updated packages then the initial code block variable reassignment issue is still present:

<img width="427" alt="screen shot 2016-09-13 at 2 48 51 pm" src="https://cloud.githubusercontent.com/assets/508936/18486917/343cac88-79c1-11e6-8dd5-835771da15d4.png">